api/client_test.go

```diff
@@ -43,3 +49,80 @@ func TestClientFromEnvironment(t *testing.T) {
 		})
 	}
 }
+
+func TestClientStreamErrorResponse(t *testing.T) {
```

Comment:
This test doesn't really test streaming since it only ever returns one response object. To correctly test stream errors, you need multiple, newline delimited responses